src/apps/bridge/sensor_drivers/aanderaaSensor.cpp

```diff
@@ -53,7 +53,22 @@ void AanderaaSensor::aanderaSubCallback(uint64_t node_id, const char *topic, uin
         uint64_t sensor_reading_time_sec = d.header.sensor_reading_time_ms / 1000U;
         uint32_t sensor_reading_time_millis = d.header.sensor_reading_time_ms % 1000U;
 
-        int8_t node_position = topology_sampler_get_node_position(node_id, 1000);
+        // Keep track of when we last got a reading from this node
+        // If it is the first reading, or if it has been more than the current aggregation period + 30 seconds
+        // then we will update the node position.
+        static int8_t node_position = 0;
+        static uint32_t last_timestamp = 0;
+        static uint32_t current_timestamp = 0;
+
+        current_timestamp = pdTICKS_TO_MS(xTaskGetTickCount());
+        if ((current_timestamp - last_timestamp > aanderaa->current_agg_period_ms + 30000u) ||
+            aanderaa->reading_count == 1U) {
+          printf("Updating rbr_coda %" PRIx64 " node position, current_time = %" PRIu32
```

Comment:
This file is named `aanderaaSensor.cpp` and this log mentions `rbr_coda`. Copy pasta?